src/components/PopupWithForm.js

```diff
@@ -23,15 +32,22 @@ export default class PopupWithForm extends Popup {
     return inputValues;
   }
 
+  // _setEventListeners() {
+  //   super._setEventListeners();
+
+  //   this._popupForm.addEventListener("submit", (evt) => {
+  //     evt.preventDefault();
+  //     this.renderLoading(true);
+  //     this._handleSubmit()
+
+  //   });
+
+  // }
+
   _setEventListeners() {
     super._setEventListeners();
-    // set submit events
+
     this._popupForm.addEventListener("submit", this._handleSubmit);
-    // this._popupForm.addEventListener("submit", (evt) => {
-    //   evt.preventDefault();
-    //   this._handleFormSubmit(this._getInputValues());
-    //   this.closeModal();
-    // });
   }
 
```

Comment:
renderLoading(isLoading, saveButtonText = "Saving...") {

Reply:
I'm not sure of the ask here. I have removed the commented-out code. renderLoading is on (now) line 54

Reply:
> I'm not sure of the ask here. I have removed the commented-out code. renderLoading is on (now) line 54

Sorry)) I wanted to say: `Good job` )) the comment wasn't saved